www-apps/rutorrent/rutorrent-3.10.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit webapp
+
+DESCRIPTION="ruTorrent is a front-end for the popular Bittorrent client rTorrent"
+HOMEPAGE="https://github.com/Novik/ruTorrent"
+SRC_URI="https://github.com/Novik/ruTorrent/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+KEYWORDS="~alpha ~amd64 ~ppc ~x86"
+IUSE=""
+
+need_httpd_cgi
+
+DEPEND="dev-lang/php[xml,gd]"
+RDEPEND="virtual/httpd-php"
+
+S="${WORKDIR}/ruTorrent-${PV}"
+
+pkg_setup() {
+	webapp_pkg_setup
+}
+
+src_prepare() {
+	default
+	find -name '\.gitignore' -type f -exec rm -rf {} \;
```

Comment:
if this gets installed I would remove it in `src_install`, does it cause build issues somehow?

Reply:
Also we generally dislike `rm -f` because it's silent about the errors and we carry out these redundant lines in ebuilds for way too long. In general you want `|| die` if *find* itself is somehow broken on your system.

You should check the source tarball. Is there truly many of these files present, or just one? Does it get installed, what's the reason for removing it here like this?